### ISSUE TITLE
`noqa: E402` not detected

### ISSUE THREAD

**Issue by spaceone** [charliermarsh/ruff#2198]
```
print(1)
from very_long_module\
    import nothing  # noqa: E402,F401
```
doesn't detect `E402`

**Comment by spaceone** [comment]
similar (but I think another issue exists for it already):
`F811` is not ignored.

```
from typing import (  # noqa: F401,F811
    overload,
)
```

**Comment by spaceone** [comment]
See also #2253

**Comment by Pierre-Sassoulas** [comment]
Similar issue:

This noqa is valid in flake8 but not detected by ruff:
```python
        is_to_run = (
            lambda x: 1  # type: ignore[assignment,misc] # pylint: disable=unnecessary-lambda-assignment
        )  # noqa: E731 We're going to throw all this anyway
```    
(While this one work for both:)
```python
        is_to_run = (  # noqa: E731, We're going to throw all this anyway
            lambda x: 1  # type: ignore[assignment,misc] # pylint: disable=unnecessary-lambda-assignment
        )
```
